packages/lesswrong/components/recommendations/RecommendationsAndCurated.jsx

```diff
@@ -181,30 +183,32 @@ class RecommendationsAndCurated extends PureComponent {
         </SubSection>
       </div>} */}
 
-      <Tooltip placement="top-start" title={curatedTooltip}>
-        <Link to={curatedUrl}>
-          <SectionSubtitle className={classNames(classes.subtitle, classes.curated)}>
-            Recently Curated
-          </SectionSubtitle>
-        </Link>
-      </Tooltip>
-      <SubSection>
-        <AnalyticsContext listContext={"curatedPosts"}>
-          <PostsList2 terms={{view:"curated", limit:3}} showLoadMore={false} hideLastUnread={true}/>
-        </AnalyticsContext>
-      </SubSection>
-      <div className={classes.footerWrapper}>
-        <Typography component="div" variant="body2" className={classes.footer}>
+      <AnalyticsContext pageSectionContext={"curatedPosts"}>
+          <Tooltip placement="top-start" title={curatedTooltip}>
```

Comment:
Incorrect indentation here, and also see note on PostsPage about making changes while preserving indent level